### PR TITLE
Proof reading changes

### PR DESCRIPTION
**Pull request by thenameiwantedwastaken**
## Description:

Minor documentation changes as a result of proofreading.

### CODE REVIEW

File: guides/faq.rst [hunk 1]
```diff
@@ -83,8 +83,8 @@ Select the firmware binary and finally press "Flash ESP".
     are connected correctly if flashing using a USB to UART bridge. For some devices you need to
     keep pressing the BOOT button until flashing has begun (ie. Geekcreit DOIT ESP32 DEVKIT V1).
 
-Help! Something's not working!1!
---------------------------------
+Help! Something's not working!!
```
Comment (OttoWinter) [inline guides/faq.rst:86]: Well this one was kinda intentional but ok :)

File: guides/getting_started_command_line.rst [hunk 2]
```diff
@@ -29,7 +29,7 @@ an RPi, please install ESPHome through ``pip`` or use :doc:`the Hass.io add-on <
 
     docker pull esphome/esphome
 
-Creating A Project
+Creating a project
```
Comment (OttoWinter) [inline guides/getting_started_command_line.rst]:
```suggestion
Creating a Project
```

Usually, the subtitles are in title-case (though I do not actively enforce that)

**Review by OttoWinter — COMMENTED** [review]
Thanks! One comment about title-case but otherwise great. 

Really thanks for going through this and actually fixing the spelling/grammar!!!

File: guides/faq.rst (hunk 1 above)
Comment (thenameiwantedwastaken) [reply guides/faq.rst:86]: Okay. I didn't get the intention... just looked like a mistake to me.

**Review by OttoWinter — APPROVED** [review]

**Pull request merged**